main/src/main/scala/sbt/Keys.scala

```diff
@@ -137,6 +137,7 @@ object Keys {
   val sbtPlugin = SettingKey[Boolean]("sbt-plugin", "If true, enables adding sbt as a dependency and auto-generation of the plugin descriptor file.", BMinusSetting)
   val printWarnings = TaskKey[Unit]("print-warnings", "Shows warnings from compilation, including ones that weren't printed initially.", BPlusTask)
   val fileInputOptions = SettingKey[Seq[String]]("file-input-options", "Options that take file input, which may invalidate the cache.", CSetting)
+  val scalaCompilerBridgeSource = SettingKey[ModuleID]("scala-compiler-bridge-source", "Configures the module ID of the sources of the compiler bridge.", CSetting)
```

Comment:
The issue is about forward compatibility here.  If Scala 2.12 comes out and requires specific sources, then sbt 0.13.x will not work anymore until a new release comes out.

I understand the reasoning, I just wanted to err on the side of forward compatibility and letting the Scala team release their own bridge-sources if needed for milestone releases.   You guys do not remember the pain-days of pre-scala-bridge-stability.